app-crypt/argon2/argon2-20160821.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Password hashing software that won the Password Hashing Competition (PHC)"
+HOMEPAGE=""https://github.com/P-H-C/phc-winner-argon2
+SRC_URI="https://github.com/P-H-C/phc-winner-argon2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( Apache-2.0 CC0-1.0 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="-static-libs"
+
+DEPEND=""
```

Comment:
The makefile adds -pthread to CFLAGS, is there a dependency that I need to specify for this? Or a USE-restraint? Otherwise, it does not link against anything: https://gist.github.com/sonOfRa/24a9997cc6cf4c5c38878d0c5222a957